python/cudf/cudf/_lib/column.pyx

```diff
@@ -438,7 +439,32 @@ cdef class Column:
             offset,
             children)
 
-    cpdef pylibcudf.Column to_pylibcudf(self):
+    # TODO: Consider whether this function should support some sort of `copy`
+    # parameter. Not urgent until this functionality is moved up to the Frame
+    # layer and made public. This function will also need to mark the
+    # underlying buffers as exposed before this function can itself be exposed
+    # publicly.  User requests to convert to pylibcudf must assume that the
+    # data may be modified afterwards.
+    cpdef pylibcudf.Column to_pylibcudf(self, mode: Literal["read", "write"]):
+        """Convert this Column to a pylibcudf.Column.
+
+        This function will generate a pylibcudf Column pointing to the same
```

Comment:
Just to note that this pattern produces, I think, badly formatted docstrings in ipython (say), since the indent is not stripped from subsequent lines.
```suggestion
        """
        Convert this Column to a pylibcudf.Column.

        This function will generate a pylibcudf Column pointing to the same
```

Reply:
Hmm could you show me a situation where it causes problems? I think the first line of docstrings are handled as a special case to support this everywhere:
```
In [1]: def f(x):
   ...:     """Hello
   ...:     
   ...:     What does this look like.
   ...:     """
   ...:     pass
   ...: 

In [3]: f?
Signature: f(x)
Docstring:
Hello

What does this look like.
File:      ~/<ipython-input-1-2dc0b815adaf>
Type:      function

In [4]: def g(x):
   ...:     """
   ...:     Hello
   ...:     
   ...:     What about this?
   ...:     """
   ...:     pass
   ...: 

In [5]: g?
Signature: g(x)
Docstring:
Hello

What about this?
File:      ~/<ipython-input-4-3369882f9e04>
Type:      function
```

Also `help(f)`:
```
Help on function f in module __main__:

f(x)
    Hello
    
    What does this look like.
```
and `help(g)`:
```
Help on function g in module __main__:

g(x)
    Hello
    
    What about this?
```

Reply:
Discussed offline with @wence- and confirmed that we don't need to make this change. I'll resolve all the associated threads as well.